crates/zoon/src/style/height.rs

```diff
@@ -39,30 +60,77 @@ impl<'a> Height<'a> {
         this
     }
 
+    /// Set the element height to fill its container or parent element.
```

Comment:
Thank you for the question. Well I get confused myself and my semantic is wrong.

A container in my old job was a complex Angular component containing other components. Sometimes it I used it to design parent element.
A parent element is an element that has child element(s).
We should stay with parent. Thank you for having pointed out my confusing writing. I will make the corrections.

It could be confusing with `html containers` as well.

Reply:
Well I am super confused about this one. I saw I used container in many cases in the documentation.

Reply:
I think both terms are interchangeable so it depends on the context. Just choose one in the associated code line to this discussion - I would probably write `container` in this case because it emphasises the visual relationship between the height of the element and the height of it's container a bit more. The parent-child relationship isn't the most important fact here.